### PR TITLE
Task/update ember 2.17

### PR DESCRIPTION
**Pull request by chuckcarpenter**
This branch updates the add-on to Ember 2.18 and supports experience builder updates

### CODE REVIEW

File: config/templates/-icon-acquia-rocket.hbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+                                  <path d="M12.5287288,9.60947368 C12.7176554,9.99684211 12.4900847,10.5094737 12.0719774,10.6373684 C11.6871469,10.7810526 11.2132203,10.5636842 11.0833333,10.18 C10.9239266,9.79421053 11.1579379,9.31421053 11.5626271,9.19105263 C11.9302825,9.05684211 12.3838136,9.24947368 12.5287288,9.60947368 Z" id="Fill-7"></path>
+                              </g>
+                          </g>
+                      </g>
+                  </g>
+              </svg>
```
Comment (ynx) [inline config/templates/-icon-acquia-rocket.hbs]: Just me being oblivious as usual - where did all the new files and hbs come from?

**Review by ynx — APPROVED** [review]
I haven't manually tested. I have left a question but nothing barring us from merging.

**Comment by julianwhiteacquia** [comment]
All looks good.

File: config/templates/-icon-acquia-rocket.hbs (hunk 1 above)
Comment (chuckcarpenter) [reply config/templates/-icon-acquia-rocket.hbs]: no, as usual, you're right we didn't need those files

**Pull request merged**